### DISCUSSION TITLE
Proper way to import views in a textual app

### DISCUSSION THREAD

**Discussion by muhamedalsayed40** [Textualize/textual#1735] [Q&A] [answered]
Hello There,

I am implementing a textual app that has 3 Dropdown selects and a button, However when the button is clicked I want the three Dropdowns to disappear and another dropdown select to appear

In this tutorial  https://dev.to/wiseai/textual-the-definitive-guide-part-3-2gl I found this
self.view.layout.docks.clear()
self.view.widgets.clear()
await self.view.dock(self.login_grid)

it's importing from textual.views
from textual.views import DockView

However when I try to do this I don't have views in __init__.py
How should I import views, so that I can add  the drop down selects as widgets to the views and then clear them using 
self.view.layout.docks.clear()
self.view.widgets.clear()

**Comment by davep** [comment, marked as answer]
The author of that blog post was working with a much older version of Textual; much has changed since that version. I'd recommend starting out with our own documentation and especially [working through the Tutorial](https://textual.textualize.io/tutorial/).
